pkgs/tools/system/jump/default.nix

```diff
@@ -0,0 +1,40 @@
+{ buildGoModule, fetchFromGitHub, lib }:
+
+buildGoModule rec {
+  pname = "jump";
+  version = "0.23.0";
+
+  src = fetchFromGitHub {
+    owner = "gsamokovarov";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "1acpvg3adcjnxnz9vx7q99cvnkkvkxfdjkbh2rb6iwakx7ksaakv";
+  };
+
+  modSha256 = "1fzsm85c31vkdw80kijxmjhk8jyhjz8b21npgks2qrnizhm6iaf8";
+
+  outputs = [ "out" "man"];
+  postInstall = ''
+    install -D --mode=444 man/j.1 man/jump.1 -t $man/man/man1/
+
+    # generate completion scripts for jump
+    export HOME=$(mktemp -d)
```

Comment:
If `HOME` is really required, please set it to $NIX_BUILD_TOP instead. See https://github.com/NixOS/nixpkgs/blob/0a1ed299f443f31bd5d0b849540002a5f15104c1/pkgs/build-support/build-bazel-package/default.nix#L22 for an example.

Reply:
Done, I cannot get it to work without `$HOME`, so I did what you suggested.